Peer/Startup.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Peer.ConnectorApi;
+using Peer.User;
+
+namespace Peer
+{
+    class Startup
```

Comment:
Startup as a class is more of a kestrel/webserver paradigm. Don't feel obligated to mimic it. Though if you want to as well you can def go to town.